experiments/utils.py

```diff
@@ -130,23 +137,24 @@ def get_alternative(self, experiment_name, request=None):
         """
         disabled = False
         if request and hasattr(request, 'experiments'):
-            disabled = experiment_name in request.experiments.disabled_experiments
+            disabled = (experiment_name in
+                        request.experiments.disabled_experiments)
 
         experiment = None
         try:
-            # catching the KeyError instead of using .get so that the experiment is auto created if desired
+            # catching the KeyError instead of using .get so that
+            # the experiment is auto created if desired
             experiment = experiment_manager[experiment_name]
         except KeyError:
             pass
         if experiment and not disabled:
             if experiment.is_displaying_alternatives():
                 alternative = self._get_enrollment(experiment)
-                if request:
-                    request.experiments_exposure = {
+                if alternative is not None:
+                    self.experiments_exposure = {
```

Comment:
Great, I think this is pretty much it. Just one more thing ™️  And really sorry to be such a PITA.

But since there can *technically* be multiple experiments active, let's make `experiments_exposure` a list here, and then let the ignoring of that fact happen in the **other PR** - there we can explicitly take the first or the last element of the list. We might even wich to make a `warning` log entry there if list has more than 1 element, so that we get alerted in Sentry when such cases occur.

Reason is that we are using this repo on multiple projects, and we are actually running multiple experiments on the same page in MT. Let's keep anything "surprising" as close to where it's needed as possible.

Reply:
Got it, makes sense 👍